docs/install/azure.rst

```diff
@@ -0,0 +1,180 @@
+your JupyterHub and configuring it, see `The Littlest JupyterHub guide <https://the-littlest-jupyterhub.readthedocs.io/en/latest/>`_.
+
+
+Step 1: Installing The Littlest JupyterHub
+==========================================
+
+We will start by creating the Virtual Machine in which we can run TLJH (The Littlest JupyterHub).
+
+#. Go to `Azure portal <https://portal.azure.com/>`_  and login with your Azure account.
+#. Expand the left-hand panel, find the Virtual Machines tab and click on it.
+
+.. image:: ../images/providers/azure/azure-vms.png
+    :alt: Virtual machines on Azure portal
+
+#. Click **+ add** to create a new Virtual Machine
+
+    .. image:: ../images/providers/azure/add-vm.png
+        :alt: Add a new virtual machine
+
+#. Select **Create VM from Marketplace** in the next sreen. This will display a new screen with all the optiond for Virtual Machines in Azure.
+    .. image:: ../images/providers/azure/create-vm.png
+        :alt: Create VM from the marketplace
+
+#. **Choose an Ubuntu server for your VM**: 
+    * Click `Ubuntu Server 18.04 LTS`
+    * Make sure `Resource Manager` is selected in the next screen and click **Create**
+
+    .. image:: ../images/providers/azure/ubuntu-vm.png
+        :alt: Ubuntu VM
+
+#. Customise the Virtual Machine basics:
+    * **Subscription**. Choose the "Free Trial" if this is what you're using. Otherwise, choose a different plan. This is the billing account that will be charged.
+    * **Resource group**. Resource groups let you bundle components that you request from Azure. If you already have one you'd like to use it select that resource.
+    * **Name**. Use a descriptive name for your virtual machine (note that you cannot use spaces or special characters).
+    * **Region**. Choose a location near where you expect your users to be located.
+    * **Availability options**. Choose "No infrastructure redundancy required".
+    * **Image**. Make sure "Ubuntu Server 18.04 LTS" is selected (from the previous step).
+    * **Authentication type**. Change authentication type to "password".
+    * **Username**. Choose a memorable username, this will be your "root" user and you'll need it later on.
+    * **Password**. Type in a password, this will be used later for admin access so make sure it is something memorable.
+
+    .. image:: ../images/providers/azure/password-vm.png
+            :alt: Add password to VM
+
+    * **Login with Azure Active Directory**. Choose "Off" (usually the default)
+    * **Inbound port rules**. Leave the defaults for now and we will update these later on in the Network configuration step.
+
+#. Before clicking on "Next" we need to select the RAM size for the image.
+    * For this we need to make sure we have enough RAM to accommodate your users. For example, if each user needs 2GB of RAM, and you have 10 total users, you need at least 20GB of RAM on the machine. It's also good to have a few GB of "buffer" RAM beyond what you think you'll need.
```

Comment:
Link to https://tljh.jupyter.org/en/latest/howto/admin/resource-estimation.html here maybe?

Reply:
This is a good suggestion, added in 51e19d